KifuSF/KFCDelivery.swift

```diff
@@ -151,14 +151,40 @@ class KFCDelivery: KFCModularTableView {
         } else if delivery.status.isAwaitingDelivery {
             
             //TODO: get location of selected charity of donation
-            return nil
+            location = CLLocationCoordinate2D(
+                latitude: delivery.latitude,
+                longitude: delivery.longitude
+            )
         } else {
             return nil
         }
         
         return KFMDestinationMap(coordinate: location)
     }
     
+    override func didSelect(_ cellType: KFCModularTableView.CellTypes, at indexPath: IndexPath) {
+        switch cellType {
+        case .destinationMap:
+            guard let delivery = self.delivery else {
+                return assertionFailure("No delivery given while map was tappable")
+            }
+            
+            if delivery.status.isAwaitingPickup {
+                
```

Comment:
nit: extra whitespace

Reply:
This is a style for me: I leave a leading line for every comment i write so I can find them better.